### PR TITLE
Proxy binary

### PR DESCRIPTION
**Pull request by mariusmihaic**
- Created proxy binary with log level for future restAPI server implementation
- Updated `go.mod`

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package main
```
Comment (iulianpascalau) [inline cmd/utils.go]: rename the file logsProcess.go ? as this is exactly what it does

File: cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+const (
+	logFilePrefix = "covalent-proxy"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Covalent proxy indexer tool"
+	app.Usage = "This is the entry point for covalent proxy indexer tool. It acts as a proxy to fetch hyperblocks from Elrond proxy. It converts hyperblocks data and and provides API calls for covalent in their desired format"
```
Comment (ssd04) [inline cmd/main.go]:
```suggestion
	app.Usage = "This is the entry point for covalent proxy indexer tool. It acts as a proxy to fetch hyperblocks from Elrond proxy. It converts hyperblocks data and provides API calls for covalent in their desired format"
```

File: cmd/config/flagsLogConfig.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package config
+
+// FLagsLog holds the log flags
+type FLagsLog struct {
```
Comment (ssd04) [inline cmd/config/flagsLogConfig.go]: maybe rename to `FlagsLog`?

File: cmd/main.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+
```
Comment (ssd04) [inline cmd/main.go]: usually we have `cmd/<app name>/main.go` and so on, maybe we should keep the same structure

File: cmd/utils.go (hunk 1 above)
Comment (mariusmihaic) [reply cmd/utils.go]: Renamed the file to `logsProcess`

File: cmd/main.go (hunk 2 above)
Comment (mariusmihaic) [reply cmd/main.go]: Right, good catch

File: cmd/config/flagsLogConfig.go (hunk 3 above)
Comment (mariusmihaic) [reply cmd/config/flagsLogConfig.go]: Good catch, renamed it

File: cmd/main.go (hunk 4 above)
Comment (mariusmihaic) [reply cmd/main.go]: Right, good catch, moved it to `cmd/proxy/main.go`

**Review by ssd04 — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**